lib/makeWebpackConfig.js

```diff
@@ -163,6 +167,7 @@ module.exports = async (playroomConfig, options) => {
         chunksSortMode: 'none',
         chunks: ['index'],
         filename: 'index.html',
+        favicon: 'images/favicon.png',
```

Comment:
Could this have come from `playroomConfig`?

Reply:
Hey David, could you log the error as an Issue with a reproduction? Thanks!

Reply:
I'm getting an error too:

```
Error: HtmlWebpackPlugin: could not load file /<path-to-project>/images/favicon.png
```